src/domains/wallet-common/env/weu-dev/terraform.tfvars

```diff
@@ -24,8 +24,12 @@ log_analytics_workspace_resource_group_name = "pagopa-d-monitor-rg"
 
 ingress_load_balancer_ip = "10.1.100.250"
 
+## DNS
+
 external_domain          = "pagopa.it"
 dns_zone_internal_prefix = "internal.dev.platform"
+dns_zone_prefix          = "dev.wallet"
```

Comment:
To update with `payment-wallet`

Reply:
Fixed in 9f2a4bf6792e5fa01f309595f1e445a690468fdd